src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1076,7 +1036,7 @@ def run_dm_activities_with_ior(self, tool, create_dataset=False, pool=None, cont
                 cont = self.container[-1]
 
         # create cont2
-        cont2 = self.create_cont(pool, oclass=self.ior_cmd.dfs_oclass.value)
+        cont2 = self.add_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'oclass' in method call